packages/central-server/src/apiV2/landingPages/EditLandingPage.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import AWS from 'aws-sdk';
+import { S3Client } from '@tupaia/utils';
+import { BESAdminEditHandler } from '../EditHandler';
+import { getStandardisedImageName } from '../../utilities';
+
+export class EditLandingPage extends BESAdminEditHandler {
+  async uploadImage(encodedImage = '', landingPageUrlSegment, type) {
+    // If the image is not a base64 encoded image, then it is already a url to an image in S3, so we can just return that url as it is. This is on the off chance that the image is being uploaded as a url instead of an encoded image, or it is not being updated at all.
```

Comment:
I've seen a couple PRs with this logic. Not sure why we didn't just pick one method and stick with it, and why we support two? (The api receiving a url or a base64 encoded image)